pages/agent/v3/elastic_ci_aws/managing_elastic_ci_stack.md

```diff
@@ -86,15 +86,27 @@ or a specific release available from the [releases page](https://github.com/buil
 
 The latest stable release can be deployed to any of our supported AWS Regions.
 
-The most recent build of the CloudFormation stack is published to
-`https://s3.amazonaws.com/buildkite-aws-stack/master/aws-stack.yml`, along with
-a version for each commit at
-`https://s3.amazonaws.com/buildkite-aws-stack/master/${COMMIT}.aws-stack.yml`.
+The most recent build of the CloudFormation stack is published to:
+
+```text
+https://s3.amazonaws.com/buildkite-aws-stack/main/aws-stack.yml
+```
+
+With a version for each commit also published at:
+
+```text
+https://s3.amazonaws.com/buildkite-aws-stack/main/${COMMIT}.aws-stack.yml
+```
+
+>📘 Versions prior to v6.0.0
+> Per commit builds for versions prior to v6.0.0, in particular for commits that are ancestors of [419f271](https://github.com/buildkite/elastic-ci-stack-for-aws/commit/419f271b54802c4c8301730bc35b34ed379074c4), were published to
+> ```text
```

Comment:
It looks like this formatting doesn't render correctly:
![Screenshot 2023-07-31 at 4 06 18 pm](https://github.com/buildkite/docs/assets/119824349/e6bd9b74-07d8-4145-9813-d569abdb98cd)

Adding a blank line seems to solve it:
![Screenshot 2023-07-31 at 4 05 53 pm](https://github.com/buildkite/docs/assets/119824349/9d186268-e059-4437-b94a-8d9c8d907a67)

```suggestion
>
> ```text
```